Î run_kubernetes.sh
Error from server (AlreadyExists): pods "machine-learning-app" already exists
NAME                   READY   STATUS    RESTARTS   AGE
machine-learning-app   1/1     Running   1          5m17s
POD status is Running
Forwarding from 0.0.0.0:8000 -> 80
Handling connection for 8000

[2020-02-25 13:27:52,073] INFO in app: JSON payload:
{'CHAS': {'0': 0}, 'RM': {'0': 6.575}, 'TAX': {'0': 296.0}, 'PTRATIO': {'0': 15.3}, 'B': {'0': 396.9}, 'LSTAT': {'0': 4.98}}
[2020-02-25 13:27:52,076] INFO in app: Inference payload DataFrame:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-02-25 13:27:52,093] INFO in app: Scaling Payload:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-02-25 13:27:52,125] INFO in app: output prediction:
{'prediction': [20.35373177134412]}
127.0.0.1 - - [25/Feb/2020 13:27:52] "POST /predict HTTP/1.1" 200 -
